docs/guides/example_messaging.md

```diff
@@ -0,0 +1,331 @@
+[**UDMI**](../../) / [**Docs**](../) / [**Guides**](./) / [Example](#)
+
+# UDMI example
+
+## Contents
+
+- [Introduction](#introudction)
+- [Physical Setup](#phyiscal-setup)
+- [Message Flow](#message-flow)
+
+## Introduction
+
+**Recommended reading**
+
+- [Tech primer](../tech_primer.md)
+- [UDMI documentation ](../)
+- [Compliance](../specs/compliance.md)
+
+This document describes a real world basic implementation of UDMI using a fan coil unit
+system as an example. Only basic messaging functionality is demonstrated and not complete functionality. 
+
+## Physical Setup
+
+For this example, the sample fan coil unit system comprises:
+- Fan
+- Cooling coil and valve
+- Supply & return air temperature sensorS
+- Fan coil unit controller
+
+![example-systems-diagram](images/example-systems-diagram.png)
+
```

Comment:
Reviewing it with your feedback, I agree I've tried to do to much and it's conflated lots of different things and added non UDMI things which is confusing for people who aren't invested in it already.

So the specific need I wanted to address was an “end to end” example and to give a reference implementation as a descriptive example rather than a piece of code. 

Currently, manufacturer engagement starts with what is essentially a summary of what I’d written here, and ends with a demonstration that their device does what has been described in this document.  And on present day UDMI use on projects, this is a very common application with so I thought having a real world application helps both system integrators on the projects and manufacturers as it

So my thoughts are:
- Call this document "HVAC Tutorial", but add to it some of the items/details I'd omitted, so that the result is a true and good  representation of one embodiment of an HVAC system with UDMI
- Create a new document which is a "messaging example" which doesn't need knowledge of anything else
- Then, add additional examples (e.g. gateway, write back) and tutorials (e.g lighting, BACnet) as required

Reply:
Hello there. I just wanted to add to this. This example you are providing is not getting into any specifics too much at all. I’m coming from controls and protocol worlds and figuring out how to make use of UDMI has been extremely frustrating and confusing. I’ve been through this repo several times trying to figure out how all the pieces connect. There should be far MORE examples like this

Reply:
I think the summary here is to take the same content and call it something like "hvac_tutorial.md" and we should be good.

Reply:
Fair enough.